frontend/src/components/Quiz/Quizzes.jsx

```
@@ -0,0 +1,178 @@
+        {currentQuestion < questions.length - 1 ? (
+          <button 
+            className="bg-blue-500 text-white px-4 py-2 rounded-md hover:bg-blue-600"
+            onClick={handleNext}
+          >
+            Next
+          </button>
+        ) : (
+          <button 
+            className="bg-green-500 text-white px-4 py-2 rounded-md hover:bg-green-600"
+            onClick={handleQuizComplete}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)